package.json

```diff
@@ -7,7 +7,7 @@
     "check-coverage": "istanbul check-coverage --statements 100 --branches 100 --functions 100 --lines 100",
     "report-coverage": "cat ./coverage/lcov.info | codecov",
     "commit": "git-cz",
-    "test": "mocha tests/**/*.js --compilers js:babel-core/register",
+    "test": "./node_modules/.bin/mocha tests/**/*.js --compilers js:babel-core/register",
```

Comment:
All executable scripts from *bin* section in  package json are linked to this .bin folder
see http://www.2ality.com/2016/01/locally-installed-npm-executables.html